template/members.tpl

```diff
@@ -22,6 +22,12 @@
             </p>
         {{ end}}
 
+        {{ if isMemberCollapsed . }}
```

Comment:
Should this be isMemberInlined

Reply:
I updated a lot of these variable and funtion names to match our naming system.